Proposed/asc058.md

```diff
@@ -0,0 +1,40 @@
+#### **Title**
+Add `attach_sourcing_info` as a datatype for the `informatics` instruction attribute
+
+#### **Authorship**
+Al Colmenar <al.colmenar@strateos.com>
+
+#### **Motivation**
+Currently, certain instructions operate on resources that don't contain the information needed to further drill down into the sourcing information associated to a particular container. This additional sourcing information is required when a customer wants to provide the vendor and supplier of the resource to be used in a run. An example of an instruction where this would be useful is the `provision` Autoprotocol instruction which is meant to describe a task of provisioning a container for a given resource.
```

Comment:
nit: I interpreted `provision` the other way:

> the `provision` Autoprotocol instruction which is meant to describe a task of provisioning a container for a given resource.

my interpretation: this descibres the task of provisioning a resource to a given container

Reply:
Ah this is true and I'll update the wording. The ASCs is more meant to address the other necessary end of the provision instruction of defining the container containing the resource to use in the run

Reply:
The containers to be used in the run are looked up based on where the resource was stored when it was checked in... So long as the resource was checked in, it'll be in the correct spot